### PR TITLE
✨feat(ui)#10271:UI - Object Storage Service Part-2

### LINKED ISSUE #10271: UI - Object Storage Service
[ObjectStorageServices.pdf](https://github.com/open-metadata/OpenMetadata/files/10801148/ObjectStorageServices.pdf)

We need to prepare the UI for the new Object Storage Service. In the mocks above, there's the page to explore the "Containers" (entities that will live inside the service).

cc @harshach @chirag-madlani

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #10271 
### Describe your changes :

I worked on #10271 and added the following things
- Details page for object store container
- Custome attribute support for container entity
- Beta tag for object store service
- Icons for object store services

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :
image

image

image

image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerChildren/ContainerChildren.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          </Link>
+        ),
+      },
+      {
+        title: t('label.description'),
+        dataIndex: 'description',
+        key: 'description',
+        render: (_, record) => (
+          <RichTextEditorPreviewer markdown={record?.description || ''} />
+        ),
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerChildren/ContainerChildren.tsx]:
```suggestion
        render: (description) => (
          <RichTextEditorPreviewer markdown={description || ''} />
        ),
```

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+  ) => {
+    const isSelectedField = editContainerColumnTags?.name === record.name;
+    const styleFlag = isSelectedField || !isEmpty(tags);
+
+    return (
+      <>
+        {isReadOnly ? (
+          <Space wrap>
+            <TagsViewer sizeCap={-1} tags={tags || []} />
+          </Space>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx:158]: I feel we should add this within <TagsViewer /> component instead parent

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+        ) : (
+          <Space
+            align={styleFlag ? 'start' : 'center'}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx:161]: Not sure what styleFlag is. Can you rename it to something meaningful?

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+        title: t('label.tag-plural'),
+        dataIndex: 'tags',
+        key: 'tags',
+        accessor: 'tags',
+        width: 350,
+        render: renderContainerColumnTags,
+      },
+    ],
+    [
+      dataModel,
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx:242]: I don't think we need this dependency here?

File: openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/CustomPropertyTable.test.tsx [hunk 5]
```diff
@@ -71,7 +72,12 @@ jest.mock('rest/metadataTypeAPI', () => ({
   ),
 }));
 
-const mockTableDetails = {} as Table & Topic & Dashboard & Pipeline & Mlmodel;
+const mockTableDetails = {} as Table &
+  Topic &
+  Dashboard &
+  Pipeline &
+  Mlmodel &
+  Container;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/CustomPropertyTable.test.tsx:80]:
```suggestion
const mockTableDetails = {} as EntityDetails;
```

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx (hunk 3 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx:161]: Sure.

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx (hunk 2 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx:158]: Thanks, I will check.

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx (hunk 4 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx:242]: Yes, it should not be there, I will remove it.

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx (hunk 2 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx:158]: @chirag-madlani, I have checked this and we have multiple references for <TagsViewer so I will take care of this in my next patch.

**Comment by Sachin-chaurasiya** [comment]
@chirag-madlani , All comments are addresses, please review.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**